runhouse/logger.py

```diff
@@ -1,7 +1,10 @@
 import logging
+import logging.config
```

Comment:
why do we import this here?

Reply:
It is used in the `get_logger()` method @jlewitt1